src/chocolatey/infrastructure.app/runners/ConsoleApplication.cs

```diff
@@ -85,9 +88,63 @@ public void run(string[] args, ChocolateyConfiguration config, Container contain
                                 }
                             }
                         },
-                        () => command.handle_validation(config),
+                        () =>
+                        {

```

Comment:
Formatting, bump this up to the end of the first line to follow consistency with the rest of these

Reply:
` () => {`